React/Views/RCTWebViewManager.m

```diff
@@ -14,6 +14,8 @@
 #import "RCTWebView.h"
 #import "UIView+React.h"
 
+#if !TARGET_OS_TV
```

Comment:
True, however these flags are only needed for certain specific components (e.g. ones that map to native iOS APIs that don't exist in tvOS).  Most components shouldn't need them.